.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["babel-plugin-transform-object-rest-spread"],
```

Comment:
Curious, did you find you had to add this after already trying with the `react` preset?

If that's the case... that kind of sucks, since babel will happily transform object spread in jsx.